src/emulator/storage/persistence.ts

```diff
@@ -74,10 +82,19 @@ export class Persistence {
   }
 
   renameFile(oldName: string, newName: string): void {
+    this._diskPathMap.set(newName, uuid.v4());
     renameSync(this.getDiskPath(oldName), this.getDiskPath(newName));
```

Comment:
do you need to move the file? it seems like just updating the pointer in diskPathMap is sufficient since we're essentially implementing a virtual file system pointer now

Reply:
Yep good catch! I'll simplify this